plugins/serialize-disabled-interactions/README.md

```diff
@@ -0,0 +1,103 @@
+# @blockly/plugin-serialize-disabled-interactions [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) plugin that for the JSON
+serialization system that serializes which are blocks are not deletable,
+movable, or editable.
```

Comment:
A Blockly plugin ~~that~~ for the JSON serialization system that serializes which ~~are~~ block are not deletable, movable, or editable.